cudax/test/utility/ensure_current_device.cu

```diff
@@ -0,0 +1,136 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Experimental in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+#define LIBCUDACXX_ENABLE_EXCEPTIONS
```

Comment:
This should already be defined in the CMakeLists.txt

Reply:
There are no redefinition errors, so it's probably not defined and the test just terminates on exception.
But I added it, since all tests need to define it anyway